iam/terraform/account-specific/main/cognito.tf

```diff
@@ -0,0 +1,39 @@
+resource "aws_cognito_user_pool" "log_viewers" {
+  name = "log_viewers"
+  password_policy {
+    minimum_length = 8
+  }
```

Comment:
We should probably specify at least as strict a policy as [the application’s policy](https://github.com/ustaxcourt/ef-cms/blob/7cab0b95e9f90ec2666e79b65c9a17fc9cb57962/web-api/terraform/template/cognito.tf#L65-L71), if not more strict.